json.md

```diff
@@ -0,0 +1,301 @@
+---
+layout: page
+title: TODO - JSON article title
+permalink: /json/
+---
+
+It is hard to think of a mobile app that doesn't need to communicate with a web server at some point. When making network-connected apps, chances are that we need to consume some good old JSON, sooner or later.
+
+In this tutorial, we look into ways of doing JSON with Flutter. We will go over what JSON solution to use in different scenarios and why.
+
+* TOC Placeholder
+{:toc}
+
+## Which JSON serialization method is right for me?
+
+There isn't a one size fits all JSON solution for every project. 
+
+Different projects come with different complexities and use cases. For smaller proof-of-concept projects or quick prototypes, using code generators might be overkill. For apps with several JSON models with more complexity, doing serialization by hand can quickly become tedious, repetitive and lend itself to lots of small errors.
+
+### Use manual serialization for smaller projects
+
+Manual JSON serialization refers to using the built-in JSON decoder in `dart:convert`. It involves passing the raw JSON string to the `JSON.decode()` method, and then looking up the values you need in the `Map<String, dynamic>` the method returns. It involves no external dependencies or particular setup process, and it is good for quick proof of concepts.
+
+Where the manual serialization does not perform well is when your project becomes bigger. Writing the serialization logic by hand can become hard to manage and error-prone. If you have a typo when accessing an unexisting JSON field, your code throws an error during runtime.
+
+If you do not have many JSON models in your project and are looking to test a concept quickly, manual serialization might be the way you want to start.
```

Comment:
link down to the example section. For example: "To learn more, and to see an example, jump to ___________"

(global comment, link to each example from these intro paragraphs)